app/model/ApplicationStatuses.scala

```diff
@@ -23,10 +23,6 @@ object ApplicationStatuses {
   val Withdrawn = "WITHDRAWN"
```

Comment:
We can add ```@deprecated``` on the top of this file to show clearly that we shouldn't add new statuses here.